test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -1371,8 +1383,10 @@ public void testInitVerifyColumnNames_sqlException() throws Exception {
   @Test
   public void testGetDocIds() throws Exception {
     executeUpdate("create table data(id integer, other varchar(20))");
-    executeUpdate("insert into data(id, other) values(1, 'hello world'),"
-        + "(2, 'hello world'), (3, 'hello world'), (4, 'hello world')");
+    executeUpdate("insert into data(id, other) values(1, 'hello world')");
```

Comment:
executeUpdate is a varargs method. Not sure whether that's more readable or not for these repetitive calls.